rest_framework/serializers.py

```diff
@@ -711,7 +711,7 @@ def to_representation(self, data):
         iterable = data.all() if isinstance(data, models.manager.BaseManager) else data
 
         return [
-            self.child.to_representation(item) for item in iterable
+            self.child.to_representation(item) if item is not None else None for item in iterable
```

Comment:
we will need extensive tests for changes

Reply:
absolutely. I didn't expect this to be merged as-is, just want to start the conversation.

In terms of tests, what kind of test do we want? I'm thinking just adding new tests for the new behavior. 

I think the tricky thing is that we don't know if the current behavior is relied upon, because there's no existing tests for that, do you know if there's a way to get that infromation?